pkg/webhook/vlanconfig/validator.go

```diff
@@ -264,8 +270,8 @@ func (v *Validator) validateMTU(current *networkv1.VlanConfig) error {
 		if vc.Name == current.Name {
```

Comment:
would it make sense to change it to just fetch the related cluster object crd and check the MTU from there?

Reply:
A few considerations:

1. The current code has assumption that the clusternetwork may be created behind the vlanconfig, for logacy or other reasons
https://github.com/harvester/network-controller-harvester/blob/ce3dd943e183e18bb42a557f02a450eb92cf1b7f/pkg/controller/manager/vlanconfig/controller.go#L86

2. The MTU is synced to clusternetwork as a label, in extreme cases, if user hack it with an invalid value, the check may miss.